gui/app/(tabs)/index.tsx

```diff
@@ -1,93 +1,296 @@
+            item={item}
+            removeCard={removeCard}
+            swipedDirection={lastSwipedDirection}
+            lastSwipedItemId={lastSwipedItemId}
+          />
+        ))}
+        {noMoreCard ? (
+          <Text style={{ fontSize: 22, color: "#000" }}>No Cards Found.</Text>
+        ) : null}
       </View>
-    </View>
+      <Text style={styles.swipeText}>
+        User Liked: {usersLiked.length}
+        User Disliked :{usersDisliked.length} 
+      </Text>
+    </SafeAreaView>
   );
-}
+};
+
+export default Swipe;
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center',
-    position: 'relative'
+    justifyContent: "center",
+    alignItems: "center",
   },
-  image: {
-    // flex: 1,
-    resizeMode: 'cover',
-    height: '100%',
-    width: '100%',
+  titleText: {
+    fontSize: 22,
+    fontWeight: "bold",
+    textAlign: "center",
+    paddingVertical: 20,
   },
-  contentsContainer: {
-    display: 'flex',
-    position: 'absolute',
-    bottom: 0,
-    left: 0,
-    padding: 10,
-    backgroundColor: 'transparent',
-    rowGap:7
+  cardStyle: {
+    width: "95%",
+    height: "70%",
+    justifyContent: "center",
+    alignItems: "center",
+    position: "absolute",
+    borderRadius: 7,
   },
-  name: {
-    fontSize: 32,
-    fontWeight: 'bold'
+  cardTitleStyle: {
+    color: "black",
+    fontSize: 24,
   },
-  distance: {
+  swipeText: {
     fontSize: 18,
-    fontWeight: 'bold'
-  },
-  status: {
-    fontSize: 16,
-    fontWeight: '400'
-    
+    textAlign: "center",
   },
-  actionsContainer: {
-    backgroundColor: 'transparent',
-    display: 'flex',
-    flexDirection: 'row',
-    width:'90%',
-    paddingHorizontal: 10,
-    justifyContent: 'space-between',
-    // borderWidth:3
-  },
-  iconContainer: {
-    backgroundColor: 'transparent',
-    width:50,
-    height:50,
-    borderRadius: 25,
-    borderWidth: 1,
-    borderColor:'#ffafcc',
-    display:'flex',
-    alignItems:'center',
-    justifyContent:'center'
-    
-  },
-  icon: {
-    
-  }
 });
+
+
+
+async function getUsersFromDb(){
+
+/*
+  getRequest('api/users').then(response => {
+    if(response.data ) {
+        return response.data
+    } else {
+      return [];
+    }
+});
+  */  
+  const DEMO_CONTENT = [
+    {
+      id: 0,
+      name: "Erica",
+      age: 28,
+      interests: ["Music", "Art", "Travel"],
+      imageUrl:
+        "https://images.unsplash.com/photo-1678875922894-7d3210b0787d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80",
+    },
+    {
+      id: 1,
+      name: "Jane",
+      age: 25,
+      interests: ["Hiking", "Food", "Reading"],
+      imageUrl:
+        "https://images.unsplash.com/photo-1678875922894-7d3210b0787d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80",
+    },
+    // add more users here...
+  ];
+  return DEMO_CONTENT; 
```

Comment:
Please use actual API endpoints for getting the data. You can seed the database manually.

Reply:
can you provide me the API URL I can fetch data from backend like we did registration form.

Reply:
Make a GET request to "/api/user/interest/{email}", where {email} is the email address of the user you are getting. Alternatively, pass the ID.

Expect a response that looks like our other API responses: 
```
{ 
  'error' => 'Error message here for display'
}
```
or 
```
{
  'success' => [
    {
      'activity' => Activity,
      'attitude' => Attitude,
      'skillLevel' => SkillLevel
    },
    {
      'activity' => Activity,
      'attitude' => Attitude,
      'skillLevel' => SkillLevel
    },
    {
      'activity' => Activity,
      'attitude' => Attitude,
      'skillLevel' => SkillLevel
    }
   ...
  ]
}
```

So you would make two requests to get the information you need: 
* One to get the profile information, such as the name, age, location, and profile picture
* One to get their interests

You can make both requests at the same time and do a fork-join on them when they come back, or make the first request to build the card, wait for it to come back, and then update the component with the results of the second request via hooks. Up to you.

If you want, we can write a method to give you all of the data in one request.